buoi9/main.js

```diff
@@ -0,0 +1,93 @@
+//bai 1:nhap luong nhan vien,tinh thue thu nhap va luong rong
+const getsalary =(salary) =>{
+    if (salary >= 15000000) {
+        console.log('Thuế thu nhập', salary * 0.3)
+        console.log('Lương ròng   ', salary * 0.7)
+    } else if (salary >= 7000000) {
+        console.log('Thuế thu nhập', salary * 0.2)
+        console.log('Lương ròng   ', salary * 0.8)
+    } else {
+        console.log('Thuế thu nhập', salary * 0.1)
+        console.log('Lương ròng   ', salary * 0.9)
+    }
+}
+//bai 2:nhap tuoi va in ra dieu kien vao lop 10
+const checkAge =(age) =>{
```

Comment:
```suggestion
const checkAge = age => {
```
nhìn kỹ cách đặt dấu cách